src/pineko/scale_variations.py

```diff
@@ -6,55 +6,149 @@
 from . import check
 
 
-def construct_ren_sv_grid(grid, max_as, nf):
-    """Generate renormalization scale variation terms for the given grid, according to the max_as.
+def ren_sv_contrib(m, delta, logpart, which_part, nf):
```

Comment:
```suggestion
def ren_sv_coeffs(m, delta, logpart, which_part, nf):
```
`contrib` has usually a different meaning in sw.